src/rdkafka_conf.h

```diff
@@ -228,6 +229,10 @@ struct rd_kafka_conf_s {
                 char *ca_location;
                 rd_kafka_cert_t *ca;
                 char *crl_location;
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+                ENGINE *ptr_engine;
```

Comment:
just call this `engine`, we don't indicate pointer in the variable name

Reply:
sure will make the change in next iteration